Assets/Scripts/Spawner/View/SpawnerView.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Zenject;
+
+public class SpawnerView : MonoBehaviour, ISpawnerView
+{
+    public event Action OnEnemyCreatingEvent;
+        
+    [SerializeField]
+    private float _spawnRadious = 2.0f;
+    [SerializeField]
+    private float _durationBetweenSpawn = 2.0f;
+    [SerializeField]
+    private int _enemysSpawnCount = 2;
+    [SerializeField]
+    private Transform _target;
+
+    private Transform _transform;
+    
+    public float SpawnRadious => _spawnRadious;
+
+    public Transform Target => _target;
+
+    public Transform Transform => _transform;
+
+    public void StartSpawner()
+    {
+        StartCoroutine(CastEnemy());
```

Comment:
Cache it and stop on destroy, and better create something like a wrapper for all coroutines to let non-monobehaviour objects use it

Reply:
Sure, then it will be more convenient to work in the presenter